gulpfile.js

```diff
@@ -81,10 +79,6 @@ gulp.task('build-sass', function(done) {
   var plugins = [
     // Autoprefix
     autoprefixer(autoprefixerOptions),
-    // Pack media queries
-    mqpacker({ sort: true }),
-    // Minify
-    cssnano(({ autoprefixer: { browsers: autoprefixerOptions }}))
```

Comment:
css-mqpacker is no longer supported -- see https://www.npmjs.com/package/css-mqpacker

cssnano is no longer needed thanks to django compressor. there _shouldn't_ be any difference between cssnano/django-compressor, and I can't see any difference in the resulting design.